client/styles/components/_hints.scss

```diff
@@ -0,0 +1,182 @@
+.CodeMirror-hints {
+  position: absolute;
+  z-index: 10;
+  overflow: hidden;
+  list-style: none;
+
+  margin: 0;
+  padding: 0;
+
+  -webkit-box-shadow: 0 0 18px 0 rgba(0, 0, 0, 0.16);
+  -moz-box-shadow: 0 0 18px 0 rgba(0, 0, 0, 0.16);
+  box-shadow: 0 0 18px 0 rgba(0, 0, 0, 0.16);
+  border: 1px solid #A6A6A6;
+
+  font-size: 100%;
+  font-family: Inconsolata, monospace;;
+
+  width: 16em;
+  max-height: 20em;
+  overflow-y: auto;
+
+  &.p5-light {
```

Comment:
You don't need to repeat the styles for each theme. You can use a SCSS mixin called `getThemifyVariable`. See [`_editor.scss`](https://github.com/processing/p5.js-web-editor/blob/develop/client/styles/components/_editor.scss)for an example for how to use it.